wolfcrypt/src/dh.c

```diff
@@ -2149,10 +2150,11 @@ int wc_DhGenerateParams(WC_RNG *rng, int modSz, DhKey *dh)
                 != MP_OKAY) {
             ret = MP_INIT_E;
         }
-    }
-    else {
-        (void)XMEMSET(&tmp, 0, sizeof(mp_int));
-        (void)XMEMSET(&tmp2, 0, sizeof(mp_int));
+        else
+        {
```

Comment:
Open parens go on the same line (wolfSSL standards)

Reply:
Sorry, I wasn't sure about that.